packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -228,8 +228,6 @@
         color: inherit;
 
         &--symbol {
-            font-size: 1.4rem;
-            line-height: 1.5;
```

Comment:
I'm replacing these repeatedly used font CSS styles with Text component in `Input` as a better practice. QA checklist will include all occurences.